v4_MIGRATION_GUIDE.md

```diff
@@ -331,6 +334,7 @@ public class GooglePayActivity extends AppCompatActivity {
       .billingAddressRequired(true)
       .googleMerchantId("merchant-id-from-google");
 
+// FEEDBACK: Why do merchants need success and error? Is there ever a case where you would have success = true and a non-null error OR success = false and a null error?
```

Comment:
Good question - there wasn't a case where we were passing back true and no error, so we updated this callback to only have an error parameter. We opened a draft PR (#346) for this change and the other migration guide fixes (and any other future feedback)